dts/arm/st/g4/stm32g474.dtsi

```diff
@@ -7,7 +7,19 @@
 #include <st/g4/stm32g473.dtsi>
 
 / {
+	aliases {
+		crc = &crc;
```

Comment:
Aliases are usually called with `0` suffix. Ie `crc0` in this case.
I agree we should not see multiple crc instances, but this is a convention used (same for watchdog, lora, ...)